File: MVP.Sample/MVP.Sample.Web/Presenters/DependencyUC1Presenter.cs
using MVP.Base.BasePresenter;
using MVP.Sample.Web.IViews;

namespace MVP.Sample.Web.Presenters
{
    public class DependencyUC1Presenter : BasePresenter<IDependencyUC1, object>
    {
        public DependencyUC1Presenter(IDependencyUC1 view) : base(view)
        {
        }

        #region Overrides of BasePresenter<IDependencyUC1,object>

        protected override object Repository
        {
            get { return null; }
        }

        protected override void ViewAction()
        {
            
        }

        protected override void UpdateAction()
        {
            
        }

        protected override void DeleteAction()
        {
            
        }

        protected override void InsertAction()
        {
           
        }

        protected override void InitializeAction()
        {
            
        }

        protected override void SearchAction()
        {
            
        }

        public override void Validate()
        {
            
        }

        #endregion
    }
}

File: DotNetNuke/Sample/Presenters/DependencyUC1Presenter.vb
Imports DNN.MVP.Base.BasePresenter
Imports DNN.MVP.Sample.ModuleName1.IViews

Namespace Presenters
    Public Class DependencyUC1Presenter
        Inherits BasePresenter(Of IDependencyUC1View, Object)

        Public Sub New(view As IDependencyUC1View)
            MyBase.New(view)
        End Sub

        Protected Overrides Sub ViewAction()
        End Sub

        Protected Overrides Sub UpdateAction()
        End Sub

        Protected Overrides Sub DeleteAction()
        End Sub

        Protected Overrides Sub InsertAction()
        End Sub

        Protected Overrides Sub InitializeAction()
        End Sub

        Protected Overrides Sub SearchAction()
        End Sub

        Public Overrides Sub Validate()
        End Sub
    End Class
End Namespace